### PR TITLE
[DOCS] Add missing config options to shared file

### PR DESCRIPTION
**Pull request by dedemorton**
@faec pointed out to me that the shared config options are not documented for Auditbeat. This PR adds a shared file that can be pulled into the module docs. I've adapted the descriptions from the ones in the Metricbeat docs. 

I've modified the docs collector script to add an asciidoc attribute to each module file. This attribute is needed to set a unique ID where the content is shared.

Questions:

* Do you think I should modify the script to add the include statement automatically? The advantage is that contributors won't overlook it; the disadvantage is that contributors will have less control over where the content appears. IMO it's better for the standard options to appear immediately after the module options.

* The module topics say,  "The following example shows all configuration options with their default values" but this isn't completely true. I see a few options (socket_type, audit_rules, audit_rule_files) that aren't shown in the example. Should they be added, or should I tweak the text?

* In auditbeat-options.asciidoc, should I conditionally exclude the `datasets` option so that it doesn't show up for modules that don't have configurable datasets? Seems like system is the only module that does, right?

### CODE REVIEW

File: auditbeat/docs/auditbeat-options.asciidoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+You can specify the following options for any {beatname_uc} module. 
+
+*`module`*:: The name of the module to run.
+
+*`datasets`*:: A list of datasets to execute. Make sure that you only list datasets that
+are available in the module. It is not possible to reference datasets from
+other modules.
+
+*`enabled`*:: A Boolean value that specifies whether the module is enabled. If you use the
+default config file, +{beatname_lc}.yml+, the System module is enabled (set to
```
Comment (cwurm) [inline auditbeat/docs/auditbeat-options.asciidoc]: These standard options are included for every module, so do we need to mention the System module explicitly?

File: auditbeat/docs/auditbeat-options.asciidoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+`enabled: true`) by default. If the `enabled` option is missing from the
+configuration block, the module is enabled by default.
+
+*`period`*:: The frequency at which the datasets check for changes. If a system
+is not reachable, {beatname_uc} returns an error for each period. This setting
+is required.
+ifeval::["{modulename}"=="system"]
+For most datasets, especially `process` and `socket`, a shorter period is recommended.
+endif::[]
```
Comment (cwurm) [inline auditbeat/docs/auditbeat-options.asciidoc]: Only the System module has `datasets` and `period` - so maybe we should only include those options for it?

File: auditbeat/docs/auditbeat-options.asciidoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+*`hosts`*:: A list of hosts to fetch information from. For some datasets, such as the
+System module, this setting is optional.
```
Comment (cwurm) [inline auditbeat/docs/auditbeat-options.asciidoc]: I don't think any of the modules or datasets in Auditbeat uses `hosts`?

File: auditbeat/docs/auditbeat-options.asciidoc (hunk 1 above)
Comment (dedemorton) [reply auditbeat/docs/auditbeat-options.asciidoc]: I think it's worth mentioning at least in the docs for the system module, so I'll make the content conditional.

File: auditbeat/docs/auditbeat-options.asciidoc (hunk 3 above)
Comment (dedemorton) [reply auditbeat/docs/auditbeat-options.asciidoc]: Removed description.

**Comment by dedemorton** [comment]
@cwurm Please confirm my changes. Thanks for your input!

**Pull request merged**